recipe/meta.yaml

```diff
@@ -8,30 +10,33 @@ package:
 source:
   url: https://github.com/flame/blis/archive/{{ version }}.tar.gz
   sha256: 729694128719801e82fae7b5f2489ab73e4a467f46271beff09588c9265a697b
-  patches:
-    # patch by upstream to solve flame/blis#514
-    - patches/0001-Attempt-to-fix-Win64-AVX512-bug.patch
 
 build:
-  number: 1
+  number: {{ build_number }}
   skip: true  # [win and vc<14]
+  string: {{ build_string }}
+  script_env:
+    - CPU_FAMILY=x86_64  # [x86_64]
+    - CPU_FAMILY=arm64   # [arm64 or aarch64]
```

Comment:
I don't know. I don't think those booleans are accessible from within the script. I'll try a few different things but I quite like the selector approach: is there anything wrong with it?

Reply:
Yes, script_env values can be overriden at test time, which we don't want.
I'd like to avoid `script_env`. Please use `${target_platform}` env variable in the build script.